util/ip.go

```diff
@@ -1,14 +1,12 @@
 package util
 
 import (
-	"fmt"
 	"io/ioutil"
 	"net/http"
 )
 
 func IP() (string, error) {
 	url := "https://api.ipify.org?format=text"
-	fmt.Printf("Getting IP address from  ipify ...\n")
```

Comment:
Maybe this could be an Info level log instead of not existing anymore

Reply:
Yes good thought!